packages/ListBox/src/components/Content/Content.js

```diff
@@ -41,25 +42,32 @@ const handleBlur = (state, dispatch) => () => {
   });
 };
 
+const handleContentFocusChange = (hasFocus, dispatch) =>
+  dispatch({ type: useListBox.types.setListBoxHasFocus, payload: { hasFocus } });
+
 export default function Content(props) {
   const [state, dispatch] = useListBox();
   const { refListBoxContainer } = state;
 
   /* NOTE no idea what ROLE should be this div when the ListBox is INLINE */
-  /* eslint-disable jsx-a11y/no-static-element-interactions */
   if (state.isInline) {
     return (
-      <div
+      <ContentStyled
```

Comment:
Currently this logic of removing the focus style of the list items when focus is moved away from the popover content is only happening when the listbox component with prop `isInline`. I ran into some issues when the listBox was simply in its basic configuration (not inline) but perhaps could be revisited if required in future.

Reply:
but is not affecting the current one correct?

Reply:
It is affecting the the listbox in the listbox browser (as both listboxes are inline there). Is that what you mean by current one?